### PR TITLE
feat/add-product-subtitle

### PR DESCRIPTION
**Pull request by edihasaj**
add product subtitle field for additional product information

### CODE REVIEW

**Comment by edihasaj** [comment]
Yes of course, I'll do that. 👍

**Comment by edihasaj** [comment]
@olivermrbl the changes are made as requested.

File: src/services/config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { QueryClient } from "react-query"
 
-let medusaUrl = "http://localhost:9000"
+let medusaUrl = process.env.API_URL || "http://localhost:9000"
```
Comment (kasperkristensen) [inline src/services/config.js:3]: rm this

File: src/services/request.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import axios from "axios"
 
-let baseURL = "http://localhost:9000"
+let baseURL = process.env.API_URL || "http://localhost:9000"
```
Comment (kasperkristensen) [inline src/services/request.js:3]: rm this

File: src/domain/products/product-form/sections/general.tsx [hunk 3]
```diff
@@ -96,6 +96,17 @@ const General = ({ showViewOptions = true, isEdit = false, product }) => {
...
+          <Input
+            id="subtitle"
+            label="Subtitle"
+            name="subtitle"
+            placeholder="Jacket, Sunglasses..."
+            ref={register()}
+            className="w-1/2"
+          />
+          <div className="w-1/2"></div>
+        </div>
```
Comment (kasperkristensen) [inline src/domain/products/product-form/sections/general.tsx:109]:
```suggestion
        <div className="grid grid-cols-2 gap-x-8 mb-base">
          <Input
            id="subtitle"
            label="Subtitle"
            name="subtitle"
            placeholder="Jacket, Sunglasses..."
            ref={register()}
          />
        </div>
```

or even better

```suggestion
<div className="grid grid-rows-3 grid-cols-2 gap-x-8 gap-y-4 mb-large">
         <Input
            id="name"
            label="Name"
            name="title"
            placeholder="Jacket, Sunglasses..."
            required
            ref={register({
              required: "Name is required",
              minLength: 1,
              pattern: /(.|\s)*\S(.|\s)*/,
            })}
          />
          <Input
            tooltipContent="Handles are human friendly unique identifiers that are appropriate for URL slugs."
            label="Handle"
            name="handle"
            placeholder="bathrobe"
            prefix="/"
            ref={register()}
          />
          <Input
            id="subtitle"
            label="Subtitle"
            name="subtitle"
            placeholder="Jacket, Sunglasses..."
            ref={register()}
          />
 </div>

```

**Review by kasperkristensen — CHANGES_REQUESTED** [review]
LGTM with some minor comments that would be nice to get in before merging

**Review by kasperkristensen — APPROVED** [review]

**Pull request merged**